cmd/OWNERS

```diff
@@ -0,0 +1,4 @@
+filters:
+  "loadTests.go":
+    approvers:
+    - perf-team
```

Comment:
Did you use "filters" because we only want to give approving power to perf-team over loadTests.go file in this directory and nothing else?

Reply:
another reason is to not let them be auto-selected as reviewers by blunderbuss plugin with regards to changes to other files in this directory. and vice-versa

Reply:
nice!